packages/core/src/server/index.ts

```diff
@@ -81,14 +75,6 @@ export async function createServer({
     }
   });
 
-  // context required for graphql middleware and hono middleware
-  const contextMiddleware = createMiddleware(async (c, next) => {
-    c.set("db", database.drizzle);
-    c.set("metadataStore", metadataStore);
-    c.set("graphqlSchema", schemaBuild.graphqlSchema);
-    await next();
-  });
-
   const hono = new Hono()
     .use(metricsMiddleware)
     .use(cors({ origin: "*", maxAge: 86400 }))
```

Comment:
Are we still validating that the user hasn't registered routes that would override the builtins? `metrics`, `health`, and `ready` ?